pkg/nodebase/NodebaseApp.js

```diff
@@ -8,9 +8,9 @@
  */
 import {App} from '../Library/App/Worker/App.js';
 import {RecipeService} from '../Library/Arcs/RecipeService.js';
-import {StoreUpdateService} from '../Library/Arcs/StoreUpdateService.js';
+import {StoreService} from '../Library/Arcs/StoreService.js';
 import {NodebaseRecipe} from './Library/NodebaseRecipe.js';
-import {LocalStoragePersistor} from '../Library/LocalStorage/LocalStoragePersistor.js';
+// import {LocalStoragePersistor} from '../Library/LocalStorage/LocalStoragePersistor.js';
```

Comment:
LocalStoragePersistor is supported.

Reply:
it wasn't used in this application before, and i'm getting an error trying to use it.... will leave as is for now.